src/tactic/spass/examples.lean

```diff
@@ -0,0 +1,115 @@
+/-
+  Copyright (c) 2019 Seul Baek. All rights reserved.
+  Released under Apache 2.0 license as described in the file LICENSE.
+  Author: Seul Baek
+-/
+
```

Comment:
I'd suggest moving this file to `/test/spass.lean`, where people expect to find the examples/tests for tactics.